liquid_node/commands.py

```diff
@@ -147,18 +142,23 @@ def deploy():
     vault.ensure_engine()
 
     vault_secret_keys = [
-        'liquid/core.django',
-        'hoover/auth.django',
-        'hoover/search.django',
-        'authdemo/auth.django',
-        'nextcloud/nextcloud.admin',
-        'nextcloud/nextcloud.maria',
-        'dokuwiki/auth.django',
-        'nextcloud/auth.django',
-        'rocketchat/auth.django',
-        'ci/vmck.django',
-        'ci/drone.secret',
+        'liquid/liquid/core.django',
+        'liquid/hoover/auth.django',
+        'liquid/hoover/search.django',
+        'liquid/authdemo/auth.django',
+        'liquid/nextcloud/nextcloud.admin',
+        'liquid/nextcloud/nextcloud.maria',
+        'liquid/dokuwiki/auth.django',
+        'liquid/nextcloud/auth.django',
+        'liquid/rocketchat/auth.django',
+        'liquid/ci/vmck.django',
+        'liquid/ci/drone.secret',
     ]
+    core_auth_apps = list(CORE_AUTH_APPS)
```

Comment:
This member should be named `auth_apps`, since the list will potentially contain apps that are not core.

Reply:
Well, the list contains *only* apps that are not core, because they authenticate against core. Hence the variable name: "apps that do auth with core".